dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/MultiReleaseJarTest.java

```diff
@@ -74,12 +76,24 @@ public static void tearDown() throws Exception {
 
     @Test
     public void testOverriddenClass_RegApp() throws Exception {
+
+        server.resetLogMarks();
+
+        assertNotNull("The application " + REGULAR_APP + " should have started by now.",
```

Comment:
Instead of doing this stuff with the log marks and waiting for specific strings in logs, we should call `server.addInstalledAppForValidation(REGULAR_APP);` after line 57 in this file (right after `ShrinkHelper.exportAppToServer(server, mrJarInWarApp);`) which will ensure `server.startServer()` blocks until all apps are ready.

We only need to add this for the `REGULAR_APP`, because `SHARED_LIB_APP` uses `ShrinkHelper.defaultApp()` which calls `addInstalledAppForValidation` under the covers, so that app is already set up OK

Reply:
Made those changes and re-pushed.